projects/assets-library/src/icons/icon-library.module.ts

```diff
@@ -58,6 +58,7 @@ const iconsRootPath = 'assets/icons';
         { key: IconType.External, url: `${iconsRootPath}/external.svg` },
         { key: IconType.ExternalApi, url: `${iconsRootPath}/external-api.svg` },
         { key: IconType.Eye, url: `${iconsRootPath}/eye.svg` },
+        { key: IconType.EyeVisibilityOff, url: `${iconsRootPath}/visibility_off.svg` },
```

Comment:
please replace '_' by a '-', ex: visibility-off,svg

Reply:
renamed the visibility icon to visibility-off.svg